modules/tinymce/src/plugins/link/main/ts/api/Options.ts

```
@@ -95,6 +99,7 @@
 const shouldShowLinkTitle = option<boolean>('link_title');
 const allowUnsafeLinkTarget = option<boolean>('allow_unsafe_link_target');
 const useQuickLink = option<boolean>('link_quicklink');
+const attributesPostProcess = option<Function>('link_attributes_postprocess');
```

Biome (1.9.4)

[error] Don't use 'Function' as a type.

Prefer explicitly define the function shape. This type accepts any function-like value, which can be a common source of bugs.

(lint/complexity/noBannedTypes)